### PR TITLE
Add hexyl

### PR DESCRIPTION
**Pull request by apcamargo**
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/hexyl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  sha256: 9c12bc6377d1efedc4a1731547448f7eb6ed17ee1c267aad9a35995b42091163
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - rust
+    - binutils              # [linux]
+    - {{native}}binutils    # [win]
```
Comment (scopatz) [inline recipes/hexyl/meta.yaml]: These two lines can just be `- {{native}}binutils`

File: recipes/hexyl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+test:
+  commands:
+    - hexyl --help
+
+about:
+  home: https://github.com/sharkdp/hexyl
+  license: MIT
```
Comment (scopatz) [inline recipes/hexyl/meta.yaml]: please add `license_family: MIT`

Comment (apcamargo) [reply recipes/hexyl/meta.yaml]: Done!

**Comment by apcamargo** [comment]
Thank you @scopatz!

**Comment by scopatz** [comment]
Weird that it fails on appveyor, but not a problem

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**